content/learn/book/getting-started/setup/_index.md

```diff
@@ -105,6 +104,15 @@ edition = "2018"
 bevy = "0.1.2" # make sure this is the latest version
 ```
 
+### Make debug builds faster
+
+Debug Rust builds are notoriously slow, and this becomes an issue when making games. Luckily, you can set up rust so your dependendencies (including Bevy) are compiled with optimizations enabled, giving you faster runtime speeds without sacrificing debuggability or iteration speed with the ```--release``` flag. To enable this, append the following to your Cargo.toml file:
+
+```toml
+[profile.dev.packages."*"]
+opt-level = 3
```

Comment:
Does this work on mac as well? I’ve found that I need to run with `debug = false` as well in order to get fast iterative compiles